src/proj_4D_api.c

```diff
@@ -480,9 +481,18 @@ Returns 1 on success, 0 on failure
         double *d = P->datum_params;
         size_t n = strlen (s);
 
+
```

Comment:
nit: I've been seeing a number of 2 blank lines inside of functions.  Is there a reason for the extra white space?

Reply:
I think in some cases it is just by mistake. In others who ever wrote the code felt that it improves clarity/readability. Hard to tell really. Since we don't have any strict formatting rules this thing happen from time to time.

Reply:
Extra line added by mistake removed